src/parser_utils.js

```diff
@@ -14,17 +14,38 @@ function childrenByName(node, name) {
 }
 
 /**
- * Parses a node text (for legacy support).
- * @param  {Object} node - The node to parse the text from.
+ * Parses a node value
+ * @param  {Object} node - The node to parse the value from.
  * @return {String}
  */
-function parseNodeText(node) {
+function parseNodeValue(node) {
+  const childNodes = node && node.childNodes && [...node.childNodes];
```

Comment:
Should we do an early return if we don't have `node` or `node.childNodes`?

Like:
```javascript
if (!node || !node.childNodes) {
  return {} // Or return ''
}
```

So we don't need to check anymore for `childNodes ` existence in the further logic